modules/apps/data-engine/data-engine-rest-impl/src/main/java/com/liferay/data/engine/rest/internal/resource/v2_0/DataDefinitionResourceImpl.java

```diff
@@ -170,7 +170,22 @@ public void deleteDataDefinition(Long dataDefinitionId) throws Exception {
 		List<DDMStructureLink> ddmStructureLinks =
 			_ddmStructureLinkLocalService.getStructureLinks(dataDefinitionId);
 
-		if (ddmStructureLinks.size() > 1) {
+		DDMStructure ddmStructure = _ddmStructureLocalService.getDDMStructure(
```

Comment:
Could you please sort these declarations as used?

Reply:
Hey @rodrigopaulino,
regarding this, the order of declaration's usage is 
`ddmStructureLinks` then `dataDefinitionContentType` and then `deDataDefinitionFieldLinks`. So, I put `ddmStructure` here because this way it is as close as possible from its unique usage, which is to define `dataDefinitionContentType`.

Do you agree with this?

Reply:
Hey, @marcosapmf. Sure, `ddmStructure` needs to be right before `dataDefinitionContentType`, but I was talking about your entire diff here, because reviewers can't comment on a block, only on lines. Sorry for not being clearer earlier.
What I meant was to sort using the following order: `deDataDefinitionFieldLinks` first, then `dataDefinitionContentType` and finally `ddmStructureLinks`. as close to its usage and [wrap everything like a burrito](https://docs.google.com/document/d/188YFP0V4n9WJIPYZL3jGBRBvEsvBNmHOU97Fdd_g6Ew/edit#heading=h.dnltur49wl4o).